k8s/modules_aws_snapshots_dev.tf

```diff
@@ -0,0 +1,23 @@
+module "test-calibrationapi" {
+  count         = local.is_dev_envs
+  source        = "../modules/cronjob_aws_snapshots_managment"
+  sts_name      = "test-calibrationapi"
+  namespace     = kubernetes_namespace_v1.network.metadata[0].name
+  creator_cron  = "*/10 * * * *"
+  deleter_cron  = "*/15 * * * *"
+  snaps_to_keep = 2
+}
+
+# module "calibrationapi_archive_snapshot" {
+#   count       = local.is_dev_envs
+#   source      = "../modules/cronjob_aws_snapshots_managment"
+#   volume_name = "calibrationapi-archive-node"
+#   namespace   = kubernetes_namespace_v1.network.metadata[0].name
+# }
+
+# module "wallaby_archive_snapshot" {
+#   count       = local.is_dev_envs
+#   source      = "../modules/cronjob_aws_snapshots_managment"
+#   volume_name = "wallaby-archive"
+#   namespace   = kubernetes_namespace_v1.network.metadata[0].name
+# }
```

Comment:
could you configure your IDE to automatically add new line in the end of the file

Reply:
Thanks for suggestion, done it :smile: